Scripts/SQLServer/DeleteByTenantKey/DeleteByTenantKey.sql

```diff
@@ -9,4 +9,11 @@ delete from [dbo].[QueueItemEvents] where TenantKey = @tenantKey;
 delete from [dbo].[QueueItems] where TenantKey = @tenantKey;
 delete from [dbo].[RobotLogs] where TenantId = @tenantId;
 delete from [read].[QueueDynamicJsonMetadata] where TenantKey = @tenantKey;
-delete from [read].[RobotDynamicJsonMetadata] where TenantKey = @tenantKey;
\ No newline at end of file
+delete from [read].[RobotDynamicJsonMetadata] where TenantKey = @tenantKey;
+
+--if you have mulitple deleted tenant, you just need to truncate read tables once
+truncate table [read].[RobotLogs];
```

Comment:
for a single tenant in a deployment with multiple tenants we also need to do delete, maybe with index rebuild : https://uipath-product.slack.com/archives/C02PW4V9S1Y/p1664920701560939?thread_ts=1664917712.632089&cid=C02PW4V9S1Y